todo_app/app.py

```diff
@@ -1,15 +1,35 @@
-from flask import Flask
-
+from flask import Flask, render_template, request, redirect, url_for
 from todo_app.flask_config import Config
 
+
+from .data import session_items as session
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    #return 'Hello World!'
+    print(session.get_items())
+    return render_template('index.html', items = session.get_items())
+
+
+@app.route('/addItem', methods=['GET', 'POST'])
+def add_item():
+    if request.method == 'POST':
+        item = request.form.get('item') 
+        print(item)
+        return item_added(item)
+    else:
+        return show_add_item_form()
```

Comment:
You have an if/else that completely splits this function in two.
You effectively have two functions, one for POST and one for GET. So why not write it like that?
```
@app.route('/addItem', methods=['POST'])
def add_item():
    item = request.form.get('item')
    return item_added(item)

@app.route('/addItem', methods=['GET'])
def add_item():
    return show_add_item_form()
```
This also supports my other comments about the item_added and show_add_item_form functions being unnecessary